dotnet/src/SemanticKernel.Abstractions/Diagnostics/ITelemetryService.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Diagnostics;
+public interface ITelemetryService
+{
+    void TrackSkillEvent(string skillName, string functionName, bool success);
```

Comment:
Additional high-level events can be added to the interface here.

Reply:
CC: @dluc / @shawncal